src/firefox/index.js

```diff
@@ -264,11 +274,12 @@ export type CopyProfileOptions = {
  */
 export async function copyProfile(
   profileDirectory: string,
+  customPrefs?: FirefoxPreferences,
```

Comment:
It makes perfect sense now. The flow error really confused me, even though that was a simple bug.